app/models/proposal.rb

```diff
@@ -143,6 +143,10 @@ def reset_status()
     end
   end
 
+  def has_subscriber?(user)
```

Comment:
I like both. `users` has parity with other method names in the model space and clearly returns an array of `User` objects. The term `subscriber` is shorthand for a business rule specific to the `Proposal` model.

Maybe `alias_method :users, :subscribers` ?